internetarchive/utils.py

```diff
@@ -86,7 +96,7 @@ def norm_filepath(fp):
 def get_md5(file_object):
     m = hashlib.md5()
     while True:
-        data = file_object.read(8192)
+        data = file_object.read(65535)
```

Comment:
why 64k instead of 8k? it would seem important to document the reason for the change at least  least in the commitlog or better, in the source code.

Reply:
Reading in larger chunks is faster.
There is a point of diminishing returns and i don't know that point but it's at least above 1/3 of hard-drive speed